src/mod_zone_difficulty_scripts.cpp

```diff
@@ -27,30 +33,71 @@ void ZoneDifficulty::LoadMapDifficultySettings()
+    HeroicTBCQuestMapList[557] = 11373;
+    HeroicTBCQuestMapList[558] = 11374;
+    HeroicTBCQuestMapList[560] = 11378;
+    HeroicTBCQuestMapList[556] = 11372;
+    HeroicTBCQuestMapList[585] = 11499;
+    HeroicTBCQuestMapList[555] = 11375;
+    HeroicTBCQuestMapList[540] = 11363;
+    HeroicTBCQuestMapList[552] = 11388;
+    HeroicTBCQuestMapList[269] = 11382;
+    HeroicTBCQuestMapList[553] = 11384;
+    HeroicTBCQuestMapList[554] = 11386;
+    HeroicTBCQuestMapList[545] = 11370;
+
+    // Icons
+    sZoneDifficulty->ItemIcons[ITEMTYPE_MISC] = "|TInterface\\icons\\inv_misc_cape_17:15|t |TInterface\\icons\\inv_misc_gem_topaz_02:15|t |TInterface\\icons\\inv_jewelry_ring_51naxxramas:15|t ";
+    sZoneDifficulty->ItemIcons[ITEMTYPE_CLOTH] = "|TInterface\\icons\\inv_chest_cloth_42:15|t ";
+    sZoneDifficulty->ItemIcons[ITEMTYPE_LEATHER] = "|TInterface\\icons\\inv_helmet_41:15|t ";
+    sZoneDifficulty->ItemIcons[ITEMTYPE_MAIL] = "|TInterface\\icons\\inv_chest_chain_13:15|t ";
+    sZoneDifficulty->ItemIcons[ITEMTYPE_PLATE] = "|TInterface\\icons\\inv_chest_plate12:15|t ";
+    sZoneDifficulty->ItemIcons[ITEMTYPE_WEAPONS] = "|TInterface\\icons\\inv_mace_25:15|t |TInterface\\icons\\inv_shield_27:15|t |TInterface\\icons\\inv_weapon_crossbow_04:15|t ";
 
     if (QueryResult result = WorldDatabase.Query("SELECT * FROM zone_difficulty_info"))
     {
         do
         {
             uint32 mapId = (*result)[0].Get<uint32>();
             uint32 phaseMask = (*result)[1].Get<uint32>();
-            ZoneDifficultyData data;
-            data.HealingNerfPct = (*result)[2].Get<float>();
-            data.AbsorbNerfPct = (*result)[3].Get<float>();
-            data.MeleeDamageBuffPct = (*result)[4].Get<float>();
-            data.SpellDamageBuffPct = (*result)[5].Get<float>();
-            data.Enabled = (*result)[6].Get<bool>();
-            sZoneDifficulty->ZoneDifficultyInfo[mapId][phaseMask] = data;
+            ZoneDifficultyNerfData data;
+            int8 mode = (*result)[6].Get<int8>();
+            if (mode == MODE_NORMAL)
+            {
+                data.HealingNerfPct = (*result)[2].Get<float>();
+                data.AbsorbNerfPct = (*result)[3].Get<float>();
+                data.MeleeDamageBuffPct = (*result)[4].Get<float>();
+                data.SpellDamageBuffPct = (*result)[5].Get<float>();
+                data.Enabled = data.Enabled | mode;
+                sZoneDifficulty->NerfInfo[mapId][phaseMask] = data;
+            }
+            else if (mode == MODE_HARD)
+            {
+                data.HealingNerfPctHard = (*result)[2].Get<float>();
+                data.AbsorbNerfPctHard = (*result)[3].Get<float>();
+                data.MeleeDamageBuffPctHard = (*result)[4].Get<float>();
+                data.SpellDamageBuffPctHard = (*result)[5].Get<float>();
+                data.Enabled = data.Enabled | mode;
+                sZoneDifficulty->NerfInfo[mapId][phaseMask] = data;
+            }
```

Comment:
I'd add a something like 
```cpp
else
{
    LOG_ERROR("module", "MOD-ZONE-DIFFICULTY: Invalid MODE used {} for mapId {}, ignored.", mode, mapId);
    return;
}
```
Incase something other than the allowed modes were entered